### PR TITLE
Adding option to run protractor testsuite with firefox

### PR DESCRIPTION
**Pull request by mcada**

### CODE REVIEW

File: ui-tests-protractor/e2e/conf/protractor.base.conf.js [hunk 1]
```diff
@@ -5,25 +5,43 @@ let reporter = require('cucumber-html-reporter');
...
 // define paths for cucumber test framework
 let testPath = path.resolve('e2e/');
 let reportsPath = path.join(testPath, 'cucumber-reports/');
 let cucumberJsonPath = path.join(reportsPath, 'cucumber-report.json');
 let cucumberHtmlPath = path.join(reportsPath, 'cucumber-report.html');
 
+let useBrowser = process.env.BROWSER || null;
 
-exports.config = {
+if (useBrowser === `firefox`) {
```
Comment (dsimansk) [inline ui-tests-protractor/e2e/conf/protractor.base.conf.js]: Pls single quotes for `'firefox'`

File: ui-tests-protractor/e2e/conf/protractor.base.conf.js [hunk 2]
```diff
@@ -5,25 +5,43 @@ let reporter = require('cucumber-html-reporter');
...
+
+  console.log(`Using firefox browser.`);
```
Comment (dsimansk) [inline ui-tests-protractor/e2e/conf/protractor.base.conf.js]: single quotes

File: ui-tests-protractor/e2e/conf/protractor.base.conf.js [hunk 3]
```diff
@@ -5,25 +5,43 @@ let reporter = require('cucumber-html-reporter');
...
+  config = { 
+    capabilities: {
+      'browserName': 'firefox'
+    }
+  };
+  
+} else {
+  
+  console.log(`Using chrome as default browser. Run \`export BROWSER=firefox\` to use firefox.`);
```
Comment (dsimansk) [inline ui-tests-protractor/e2e/conf/protractor.base.conf.js]: And one more... Maybe proper capitalization for browser names. My OCD kicks-in sorry :)

**Comment by dsimansk** [comment]
@mcada I forgot to check protractor version. I suppose we still need to upgrade to `5.2.0` to run Firefox properly, or not?

**Comment by mcada** [comment]
It works now even with version 5.1.1, but I am not sure why it didn't work last week :-/ and I am sure it did not...

**Pull request merged**